.test-infra/jenkins/job_PostCommit_Python_Sickbay.groovy

```diff
@@ -19,10 +19,10 @@
 import CommonJobProperties as commonJobProperties
 import PostcommitJobBuilder
 
-import static PythonTestProperties.ALL_SUPPORTED_VERSIONS
+import static PythonTestProperties.ESSENTIAL_VERSIONS
```

Comment:
All_SUPPORTED_VERSIONS sounds right in this context. Why change that?

Reply:
https://ci-beam.apache.org/view/PostCommit/job/beam_PostCommit_Sickbay_Python310/ never worked and no one find it for months. That led to me think about using ESSENTIAL_VERSIONS suffices.

```
08:45:45 Task 'python310SickbayPostCommit' not found in root project 'beam'. Some candidates are: 'python37SickbayPostCommit', 'python38SickbayPostCommit', 'python39SickbayPostCommit'.
```

Reply:
This calls out for an issue tracker that could monitor and file github issues for these long unhealthy tests. 

Anyways, I am in support of keeping ALL_SUPPORTED_VERSIONS since we use this list to get lowest and highest python versions. Essential sounds like `good to have`

Reply:
changed back to ALL_SUPPORTED_VERSIONS. Seed job passed: https://ci-beam.apache.org/job/beam_SeedJob/11388/